packages/flutter/test/cupertino/nav_bar_test.dart

```diff
@@ -157,6 +157,32 @@ void main() {
     );
   });
 
+  testWidgets('Blur affect is disabled when enableBackgroundFilterBlur is false', (WidgetTester test) async {
```

Comment:
I think this test would pass even without the rest of the PR, if a background color without any transparency is used for the NavBar.

Reply:
Bumping on this, the test should pass in a background color with some transparency.